content/v1.12/getting-started/provider-azure-part-2.md

```diff
@@ -0,0 +1,1050 @@
+
+```shell {copy-lines="1"}
+kubectl get linuxvirtualmachine
+NAME                                 READY   SYNCED   EXTERNAL-NAME                        AGE
+my-composite-resource-w564c          True    True     my-composite-resource-w564c          16m
+my-second-composite-resource-s92lw   True    True     my-second-composite-resource-s92lw   5m8s
+```
+
+```shell {copy-lines="1"}
+kubectl get networkinterface
+NAME                                 READY   SYNCED   EXTERNAL-NAME                        AGE
+my-composite-resource-72ft8          True    True     my-composite-resource-72ft8          16m
+my-second-composite-resource-wcnnv   True    True     my-second-composite-resource-wcnnv   5m21s
+```
+
+### Delete the composite resources
+Because the _composite resource_ is the `Owner` of the _managed resources_, when
+Crossplane deletes the _composite resource_, it also deletes the _managed resources_ automatically.
+
+Delete the new _composite resource_ with `kubectl delete xvirtualmachine`.
+
+{{<hint "tip" >}}
+Delete a specific _composite resource_ with `kubectl delete <kind>` or
+`kubectl delete composite`.
+{{< /hint >}}
+
+Delete the second composition
+```shell
+kubectl delete xvirtualmachine my-second-composite-resource
+```
+
+{{<hint "note">}}
+It may take up to five minutes before Crossplane finishes deleting resources.
+{{</hint >}}
+
+Now only one virtual machine and network interface exist.
+
+```shell {copy-lines="1"}
+kubectl get linuxvirtualmachines
+NAME                          READY   SYNCED   EXTERNAL-NAME                 AGE
+my-composite-resource-w564c   True    True     my-composite-resource-w564c   28m
+```
+
+```shell {copy-lines="1"}
+kubectl get networkinterface
+NAME                          READY   SYNCED   EXTERNAL-NAME                 AGE
+my-composite-resource-72ft8   True    True     my-composite-resource-72ft8   29m
+```
+
+Delete the other _composite resource_ to remove the last `linuxvirtualmachines` 
+and `networkinterface` _managed resources_.
+
+```shell
+kubectl delete xvirtualmachine my-composite-resource
+```
+
+_Composite resources_ are great for creating one or more related resources against
+a template, but all _composite resources_ exist at the Kubernetes "cluster
+level." There's no isolation between _composite resources_. Crossplane uses
+_Claims_ to create resources with namespace isolation. 
+
+## Create a claim
+
+_Claims_, just like _composite resources_ use the custom API defined in the
+_XRD_. Unlike a _composite resource_, Crossplane can create _Claims_ in a
+namespace.
+
+### Create a new Kubernetes namespace
+Create a new namespace with `kubectl create namespace`.
+
+```shell
+kubectl create namespace test
+```
+
+Look at the _XRD_ to see the parameters for the _Claim_.
+A _Claim_ uses the same {{<hover label="XRDclaim2" line="6" >}}group{{</hover>}}
+a _composite resource_ uses but a different 
+{{<hover label="XRDclaim2" line="8" >}}kind{{</hover>}}.
+
+```yaml {label="XRDclaim2"}
```

Comment:
this block shows a copy clipboard icon, but i'm pretty sure is not intended to be copied - can that be removed?

Reply:
fixed